### DISCUSSION TITLE
Is there any way to disable SSR overriding base path?

### DISCUSSION THREAD

**Discussion by AlejandroAkbal** [nuxt-alt/auth#106] [Q&A] [answered]
My use case: I have a Nuxt 3 application that uses hybrid mode to prerender some pages, but the rest are normal spa
But to prerender you need ssr: true in the nuxt config

When enabling ssr: true, this package overrides the base URL to local, which is wrong in my case, since I am still using a separate backend for authenticating with refresh scheme
https://github.com/nuxt-alt/auth/blob/ea5d9b8aef8205ba3732284627497ce0aab372a8/src/runtime/schemes/refresh.ts#L157

I would love to make this changeable in the config
https://github.com/nuxt-alt/auth/blob/ea5d9b8aef8205ba3732284627497ce0aab372a8/src/resolve.ts#L33

So it would not override the base URL override, and it would work the same as ssr: false

**Comment by Denoder** [comment, marked as answer]
Try using `@nuxt-alt/auth@beta` created a draft for you to follow if you have any changes you need regarding that property: https://github.com/nuxt-alt/auth/pull/107

**Reply by AlejandroAkbal** [reply]
Damn that was fast, thank you Denoder, love you!

**Comment by nwaenwae** [comment]
Hello @Denoder, may I know when will the 3.1.7 stable version be released? I encountered the SSR overriding base path issue with version 3.1.6 so that I tried to use 3.1.7 beta version. It works perfectly. Now I am working on a project by using this nuxt-alt/auth so that I would like to know when it will be released.
